src/worker/install/install.sh

```diff
@@ -37,10 +38,12 @@ then
         # Install open telemetry related packages
         python3 -m pip -qqq install opentelemetry-sdk opentelemetry-exporter-otlp
         
-        # Pull Geneva Metrics Extension(MA) docker image
-        docker pull linuxgeneva-microsoft.azurecr.io/genevamdm:$MDM_DOCKER_VERSION
-        docker tag linuxgeneva-microsoft.azurecr.io/genevamdm:$MDM_DOCKER_VERSION genevamdm
-        docker rmi linuxgeneva-microsoft.azurecr.io/genevamdm:$MDM_DOCKER_VERSION
+        if ! sudo docker images | grep genevamdm | grep -q $DOCKER_IMAGE_ID; then
+            # Pull Geneva Metrics Extension(MA) docker image
+		    docker pull linuxgeneva-microsoft.azurecr.io/genevamdm:$MDM_DOCKER_VERSION
```

Comment:
Was this code block causing issues if the image was already pulled down? Idk if I want to hard code an image ID because this could change on a new iteration of the docker file,

Reply:
Maybe just this: ```if sudo docker images |  grep -q  genevamdm ; then```